.github/workflows/dart_ci.yaml

```diff
@@ -12,21 +12,17 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        sdk: [ 2.18.7, 2.19.6 ]
     steps:
       - uses: actions/checkout@v2
       - uses: dart-lang/setup-dart@v1
         with:
-          sdk: ${{ matrix.sdk }}
+          sdk: 2.19.6
       - name: Install dependencies
         run: dart pub get
       - name: Validate dependencies
         run: dart run dependency_validator
-      - name: Check formatting
-        run: dart format --output=none --set-exit-if-changed .
-        if: ${{ matrix.sdk == 'stable' }}
+      # - name: Check formatting
+      #   run: dart format --output=none --set-exit-if-changed .
```

Comment:
Formatting hasn't been ran on the repo in awhile due to the `matrix.sdk == 'stable'` check

Will put up a different PR, running formatting, so as to not obscure the changes here with formatting diffs